src/fastoad/notebooks/02_explanation/05_FAST-OAD.ipynb

```diff
@@ -0,0 +1,441 @@
+{
```

Comment:
<p>Maybe be more precise regarding bricks (OpenMDAO components)?</p><blockquote>decoractor</blockquote><p>decorator</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Changed in dfe233f7